docs/2-reference_guide/reference_guide.md

```diff
@@ -1,207 +1,225 @@
 # Reference guide for the entanglement forging module
 
 ## Table of Contents
+1. [Installation instructions](#installation-instructions)
+    - [Basic installation](#basic-installation)
+    - [Installation from source](#installation-from-source)
+2. [Using the module](#using-the-module)
+    - [Specifying the problem](#specifying-the-problem)
+    - [Specifying the bitstrings](#specifying-the-bitstrings)
+    - [Freezing orbitals](#freezing-orbitals)
+    - [Specifying the Ansatz](#specifying-the-ansatz)
+    - [Options (`EntanglementForgedConfig`)](#options-entanglementforgedconfig)
+    - [Specifying the converter](#specifying-the-converter)
+    - [The solver](#the-solver)
+    - [Running the algorithm](#running-the-algorithm)
+    - [Viewing the results](#viewing-the-results)
+    - [Verbose](#verbose)
+3. [Troubleshooting](#troubleshooting)    
+    - [Getting poor results on the hardware](#getting-poor-results-on-the-hardware)
+    - [For IBM Power users](#for-ibm-power-users)    
+
+This guide is for those who just want to use the package. If you want to extend the module or documentation, read [this other guide](CONTRIBUTING.md) instead. Installation instructions are only located here to avoid repetition.
+
+
+## Installation instructions
+:exclamation: _This prototype depends on the PySCF package, which does not support Windows; therefore, Windows users will not be able to install and use this software. Advanced Windows users may wish to attempt to install PySCF using Ubuntu via the Windows Subsystem for Linux.  We are exploring the possibility of providing Docker support for this prototype so it can be used within Docker Desktop, including on Windows._
```

Comment:
Does it make sense to have this warning in the contribution guide as well?

Reply:
The contribution guide links directly to the reference guide for installation instructions; so, personally, I would avoid repetition.